docs/CreateAKSClusterForScalarDLAuditor.md

```diff
@@ -0,0 +1,215 @@
+# Create an ASK cluster for ScalarDL Ledger and ScalarDL Auditor
+
+This document explains how to create an ASK cluster for the ScalarDL Ledger and ScalarDL Auditor deployment. For details on how to deploy ScalarDL Ledger and ScalarDL Auditor on the ASK cluster, please see [Deploy ScalarDL Ledger and ScalarDL Auditor on AKS (Azure Kubernetes Service)](./ManualDeploymentGuideScalarDLAuditorOnAKS.md).
+
+## Steps
+
+You must create an AKS cluster based on the following requirements, recommendations, and your project's requirements. Please refer to the Azure official document ([Azure CLI](https://learn.microsoft.com/ja-jp/azure/aks/learn/quick-kubernetes-deploy-cli), [PowerShell](https://learn.microsoft.com/ja-jp/azure/aks/learn/quick-kubernetes-deploy-powershell), or [Azure portal](https://learn.microsoft.com/ja-jp/azure/aks/learn/quick-kubernetes-deploy-portal)) for the details about how to create the AKS cluster.
+
+## Requirements
+
+There are some requirements for the deployment of ScalarDL Ledger and ScalarDL Auditor. Please configure your ASK cluster based on the following requirement and your project's requirements.
+
+* You must create the ASK cluster with Kubernetes version 1.21 or higher for the ScalarDL Ledger and ScalarDL Auditor deployment.
+* You cannot deploy your application pods on the same ASK cluster as ScalarDL Ledger and ScalarDL Auditor deployment to make Byzantine fault detection work properly.
+* You must create two ASK clusters. One is for ScalarDL Ledger and another one is for ScalarDL Auditor.
+* You must configure a VNet as follows.
```

Comment:
In the ScalarDL Auditor mode document, users have to connect between two Vnets to keep administrative domains separate.